README.md

```diff
@@ -146,6 +148,23 @@ Another example is the tutorial for the SARS-CoV-2 workflow, which lives alongsi
 Subprojects necessitate a distinct URL, such as https://docs.nextstrain.org/projects/augur as opposed to just https://docs.nextstrain.org.
 By default, this means a distinct set of headings / links in the sidebar navigation, making navigating back to https://docs.nextstrain.org more difficult once you have navigated to a subproject.
 
+## Building the SubProject docs
+
+Typically the documentation for each subproject is build using the same commands as the umbrella docs described above:
+```
+cd <repo>/docs
+make clean # optional
+make html
+```
+This may work in the `docs.nextstrain.org` conda environment, but some subprojects require additional dependencies.
+For full details, see the respective how-to-build-docs sections:
+
+- [Augur](https://github.com/nextstrain/augur/blob/master/docs/contribute/DEV_DOCS.md#building-documentation)
+- [Auspice](https://github.com/nextstrain/auspice/blob/master/DEV_DOCS.md#contributing-to-documentation)
+- [Nextclade (includes Nextclade Web, Nextclade CLI, Nextalign CLI)](https://github.com/nextstrain/nextclade/tree/master/docs#building-the-docs-locally)
+- [ncov (SARS-CoV-2 Workflow)](https://github.com/nextstrain/ncov/tree/master/docs#building-the-docs)
```

Comment:
Not something to address with this PR, but seeing these links to GitHub instead of https://docs.nextstrain.org makes me think we'd be better off in the long run making our dev docs (including this README here) part of https://docs.nextstrain.org and using cross-refs. Helps make the links more robust to breaking and makes it clear if they do break, while also surfacing more of our dev docs.

Reply:
👍